s2p/geographiclib.py

```diff
@@ -80,44 +80,51 @@ def epsg_code_from_utm_zone(utm_zone):
     return const + zone_number
 
 
-def utm_proj(utm_zone):
+def crs_proj(projparams):
```

Comment:
This function seems a bit redundant now that `pyproj_crs()` exists, I think we could get rid of it.

It is only used in `s2p.rasterization.plyflatten_from_plyfiles_list()`, where the `srs` attribute of the returned `Proj` object is used when writing a GeoTIFF with `rasterio`.

An easy change could be to use in `s2p.rasterization.plyflatten_from_plyfiles_list()` a `pyproj.CRS` object returned by `pyproj_crs()` instead of a `pyproj.Proj` object returned by `crs_proj()`.

The `srs`  attributes of these objects are not exactly the same, as can be seen here:
```python
In [1]: import pyproj

In [2]: pyproj.Proj("epsg:32740").srs
Out[2]: '+proj=utm +zone=40 +south +datum=WGS84 +units=m +no_defs'

In [3]: pyproj.CRS("epsg:32740").srs
Out[3]: 'epsg:32740'
```
however, I think it should work well enough with rasterio/GDAL.

If we wanted to do things really by the book, however, I think we should follow what is said in the pyproj documentation's section [Converting from pyproj.crs.CRS to rasterio.crs.CRS](https://pyproj4.github.io/pyproj/stable/crs_compatibility.html#converting-from-pyproj-crs-crs-to-rasterio-crs-crs):
```python
from distutils.version import LooseVersion

import rasterio
from rasterio.crs import CRS as RioCRS
from pyproj.crs import CRS
from pyproj.enums import WktVersion

proj_crs = CRS.from_epsg(4326)
if LooseVersion(rasterio.__gdal_version__) < LooseVersion("3.0.0")
    rio_crs = RioCRS.from_wkt(proj_crs.to_wkt(WktVersion.WKT1_GDAL))
else:
    rio_crs = RioCRS.from_wkt(proj_crs.to_wkt())
```

Reply:
I took your comment into account and proposed this implementation: 6001eb6d039ac842560985e95f26330a825ecdb2